README.md

```diff
@@ -75,12 +75,12 @@ bookshelf.Model.extend({
 
 Contributions are welcome and greatly appreciated, so feel free to fork this repository and submit pull requests.  
 
-**bookshelf-json-columns** supports PostgreSQL and SQLite3. You can find test suites for each of these database engines in the *test/postgres* and *test/sqlite* folders.
+**bookshelf-json-columns** supports PostgreSQL, SQLite3 and MySQL. You can find test suites for each of these database engines in the *test/postgres*, *test/sqlite* and *test/mysql* folders.
 
 ### Setting up
 
 - Fork and clone the **bookshelf-json-columns** repository.
-- Duplicate *test/postgres/knexfile.js.dist* and *test/sqlite/knexfile.js.dist* files and update them to your needs.
+- Duplicate *test/postgres/knexfile.js.dist*, *test/sqlite/knexfile.js.dist* and *test/mysql/knexfile.js.dist* files and update them to your needs.
```

Comment:
Maybe, in relation to the comment above: `Duplicate each *.dist* database engine test file and update them to your needs.`